vscode-antimony/src/server/stibium/stibium/analysis.py

```diff
@@ -430,6 +440,19 @@ def handle_annotation(self, scope: AbstractScope, annotation: Annotation):
         self.table.insert(qname, SymbolType.Parameter)
         self.table.insert_annotation(qname, annotation)
     
+    def pre_handle_sboterm(self, scope: AbstractScope, sboterm: Sboterm):
+        self.pending_sboterms.append((scope, sboterm))
+    
+    def handle_sboterm_list(self):

```

Comment:
Is method internal to the class? Should start with an "_". If this is external, you should document what the function does. Also, what does "handle" mean? Is there a more descriptive verb?